service/src/main/java/uk/gov/hmcts/reform/fpl/model/CaseData.java

```diff
@@ -34,14 +37,30 @@
     @NotNull(message = "You need to add details to orders and directions needed")
     @Valid
     private final Orders orders;
+
+    public boolean hasEPOGrounds() {
+        return orders != null && orders.getOrderType() != null
+            && orders.getOrderType().contains(OrderType.EMERGENCY_PROTECTION_ORDER);
+    }
+
     private final Grounds grounds;
     private final GroundsForEPO groundsForEPO;
     @NotNull(message = "You need to add details to children")
     @Valid
     private final Children children;
-    @NotNull(message = "You need to add details to applicant")
+    private final OldApplicant applicant;
+    private final List<Element<Applicant>> applicants;
+
     @Valid
-    private final Applicant applicant;
+    public final Applicant getMainApplicant() {
+        if (applicants != null && applicants.get(0) != null && applicants.get(0).getValue() != null) {
```

Comment:
So my only problem is that, if we omit first applicant and provide details for the second one, the validation will fail, is it something business is happy with? I guess it's quite an edge case so shouldn't be a problem but probably worth checking.

Reply:
Yeah true, I wasn't convinced by this either. I guess worst case scenario is they won't get applicants validated. I'll check with Theo and updated with what he says.

Reply:
Theo has stated to not worry about it as a story will be written to make applicants non collection. Tech debt for now unfortunately.

Reply:
Replace with annotations on applicants collection. I also do not like returning empty applicant if applicants collection is empty.

Here is an example how to validate elements of the collections: https://www.baeldung.com/bean-validation-container-elements

Reply:
We only need to validate the first applicant in the collection, adding validations to entire collection would repeat validation messages which we do not want.

Reply:
Why first only? Why not to apply it to every applicant, they will only use one.